projectile.el

```diff
@@ -43,6 +43,7 @@
 (require 'compile)
 (require 'grep)
 (require 'lisp-mnt)
+(require 'xml)
```

Comment:
I'd suggest moving this in the function that uses it, so we don't slow down loading Projectile needlessly.